### PR TITLE
Assistant is now only for new players, and when all other job slots are filled.

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

Disables picking assistant if you're not new to the game, and there's still job slots open.

Made at the request of oranges.

## Why It's Good For The Game

Made at the request of oranges.
from 4 years ago:
uCkRIkP
from yesterday:
assistant1
assistant3
assistant2

we really shoulda done this ages ago, @imtakingabreakdontatme was right on the money

## Changelog
:cl:
balance: Assistant is now unavailable to players who aren't new to the game, and job slots are still open.
/:cl:

### CODE REVIEW

**Comment by joookschad** [comment]
retard

**Comment by FernandoJ8** [comment]
Fuck assistants.
You're a hero.

**Comment by Bluedazzled** [comment]
the greytide must live on, terrible pr

**Comment by rdragan** [comment]
lol

**Comment by tralezab** [comment]
this is going to change a lot about how assistants are viewed

**Comment by wesoda25** [comment]
So what role do I go when I don't want to do a job

**Comment by joookschad** [comment]
> So what role do I go when I don't want to do a job

just dont play :trollface:

**Comment by celotajstg** [comment]
Unironically seems like a change worth testing, seriously.

I believe it is worth test-merging and seeing how it goes.

**Comment by tralezab** [comment]
@wesoda25 there are a lot of open ended jobs with no real goals right now like chaplain, psych, curator, and lawyer.

**Comment by necromanceranne** [comment]
> So what role do I go when I don't want to do a job

Lawyers, curators and psychologists don't really have a job that they do, and chaplains are a good role if the extent of what you want to do is buddy up to fight shit. 

Most cargo techs can faff about. Engineers have considerable freedom. Scientists mostly piss about. The idea of not doing a job is absurd, since as an assistant you are probably doing _something_. And if you're not doing anything at all, then...

Iunno.

**Comment by SamDDBS** [comment]
just because oranges said its good for the game doesnt mean it is

**Comment by wesoda25** [comment]
And if I don't want to explore the options available to those jobs? I'll just be depriving the role from someone who does

**Comment by NanoCats** [comment]
This is dumb, Assistant is a good role when I want to fart around/explore space/etc without any huge responsibilities. This will literally just make people pick whatever's open and neglect their duties.  The clowns on staff are supposed to make people laugh not write PRs.

**Comment by SinfulBliss** [comment]
The sec-assistant dynamic gives life to the game. Without it it's just kind of each department to themselves, working alone. There's no little pranks, there's no officers getting slipped and losing their batons, there's no fun harmless combat.

All you will have now is people forced to choose a job, who don't actually want to play that job, so they just tide with an engi ID or whatever it is. Not to mention all the assistant-specific features this will break.

**Comment by SuperNovaa41** [comment]
> 
> 
> So what role do I go when I don't want to do a job

roll clown or mime and use assistant as fallback

**Comment by necromanceranne** [comment]
>Not to mention all the assistant-specific features this will break.

a smelly jumpsuit is not really a feature. And who cares about the tc toolbox, it's just the cowards his grace.

**Comment by SuperNovaa41** [comment]
> 
> 
> > Not to mention all the assistant-specific features this will break.
> 
> a smelly jumpsuit is not really a feature. And who cares about the tc toolbox, it's just the cowards his grace.

tc toolbox is fucking awesome

**Comment by celotajstg** [comment]
> This is dumb, Assistant is a good role when I want to fart around/explore space/etc without any huge responsibilities. This will literally just make people pick whatever's open and neglect their duties. The clowns on staff are supposed to make people laugh not write PRs.

actually good point

**Comment by bluetshirtguy** [comment]
your pr isn't very descriptive

**Comment by SuperNovaa41** [comment]
> 
> 
> your pr isn't very descriptive

how so? he describes exactly whats changing

File: code/controllers/subsystem/job.dm [hunk 1]
```diff
@@ -323,6 +323,9 @@ SUBSYSTEM_DEF(job)
 	var/list/overflow_candidates = FindOccupationCandidates(overflow, JP_LOW)
 	JobDebug("AC1, Candidates: [overflow_candidates.len]")
 	for(var/mob/dead/new_player/player in overflow_candidates)
+		if(player.IsJobUnavailable(SSjob.overflow_role) != JOB_AVAILABLE)
+			JobDebug("AC1 failed, removing from overflow pool, Player: [player]")
+			overflow_candidates -= player
 		JobDebug("AC1 pass, Player: [player]")
```
Comment (TiviPlus) [inline code/controllers/subsystem/job.dm:329]:
```suggestion
		if(player.IsJobUnavailable(SSjob.overflow_role) != JOB_AVAILABLE)
			JobDebug("AC1 failed, removing from overflow pool, Player: [player]")
			overflow_candidates -= player
			continue
		JobDebug("AC1 pass, Player: [player]")
```

**Comment by SuperNovaa41** [comment]
I think there's probably a better way to do this, but this is probably a good temporary solution... hopefully

**Comment by AtlasRules** [comment]
it's fucking over

leave SS13 while you can it's been downhill since 2018 but this is it

**Comment by Iamgoofball** [comment]
i put some justification in the OP

**Comment by AtlasRules** [comment]
> 
> 
> Fuck assistants.
> You're a hero.

mad because bad

**Comment by SplinterGP** [comment]
time to greytide with other jobs

**Comment by OmegaDarkPotato** [comment]
> > So what role do I go when I don't want to do a job
> 
> Lawyers, curators and psychologists don't really have a job that they do, and chaplains are a good role if the extent of what you want to do is buddy up to fight shit.
> 
> Most cargo techs can faff about. Engineers have considerable freedom. Scientists mostly piss about. The idea of not doing a job is absurd, since as an assistant you are probably doing _something_. And if you're not doing anything at all, then...
> 
> Iunno.

do we really want to further encourage taking a job slot when you don't want to actually work as that job?
I'd much rather assistant stay in than take the job slot of a medical doctor when I just want to make maint medical and fuck off to make drugs the whole round.

**Comment by AtlasRules** [comment]
> if(isnum(client.player_age) && client.player_age <= 14) //Newbies can always be assistants

this game takes significantly longer than that to learn believe you me

**Comment by Iamgoofball** [comment]
@AtlasRules By that point you've learned how to operate your inventory, walk around, and do basic tasks, making you qualified for Cargo Tech, Medical Doctor, Engineer, etc.

**Comment by AtlasRules** [comment]
> 
> 
> @AtlasRules By that point you've learned how to operate your inventory, walk around, and do basic tasks, making you qualified for Cargo Tech, Medical Doctor, Engineer, etc.

absolutely fucking not lmfao

**Comment by tralezab** [comment]
considering with this change they'll never be able to have assistant again it could definitely be increased, fo sho

**Comment by ardentarclight** [comment]
Why not add a set ratio of assistants to actual jobs, preventing more assistants joining and giving random jobs when the amount of assistants reaches the limit?

**Comment by NanoCats** [comment]
> qualified for Cargo Tech, Medical Doctor, Engineer, etc.

The Fat Controller laughed. "You are wrong."

**Comment by SuperNovaa41** [comment]
> 
> 
> Why not add a set ratio of assistants to actual jobs, preventing more assistants joining and giving random jobs when the amount of assistants reaches the limit?

this would probably be a way better solution tbh, or even just a fucking pop limit

**Comment by gorillion** [comment]
this has already been said, but all this is going to do is make those assistant players pick jobs and completely neglect their duties and just go tide as they normally would, which is already an issue.
what if, instead, admins went and made a new rule that made it so people in "important" jobs (anything outside of assistant/rp jobs) had to at least make an attempt to fill their role?

**Comment by Gannets** [comment]
Godspeed.

**Comment by SaltyOldBull** [comment]
This seems like a mistake.

**Comment by AtlasRules** [comment]
> 
> 
> This seems like a mistake.

no really

**Comment by AtlasRules** [comment]
image
extremely relevant meme

**Comment by necromanceranne** [comment]
> do we really want to further encourage taking a job slot when you don't want to actually work as that job?
> I'd much rather assistant stay in than take the job slot of a medical doctor when I just want to make maint medical and fuck off to make drugs the whole round.

If that's the goal for the round, medical is the role to take. You've got a lot of the groundwork covered, and who said that _can't_ be what you do for the round? If you're worried about not contributing to the role's productivity or effectiveness, nobody expects you to be good at your job. You're only expected to not be a negative impact upon the experience of other players, which is mutually exclusive to not being a perversion of your roles responsibilities in a mostly harmless way.

You weren't going to do nothing in the round, so it'd be best to pick the role that lets you springboard into what you want to do. In fact, if you did this as assistant, isn't this somewhat encroaching upon the job of people in dedicated roles, like botany/chemistry/medical?

**Comment by itseasytosee** [comment]
People really getting rilled up about a goof shit post.

**Comment by ExcessiveUseOfCobblestone** [comment]
seems like an admin problem that they aren't going to address even with the change which is having people fuck off and solely play to kill or be killed without the normal restrictions of security.

I would say this would be a good feature if it had admin backing to ensure people are doing whatever the change is expecting them to do afterwards. Whats the actual change if the same number of people are doing what they would have done as assistant if there's no expectation on administration afterwards?

**Comment by Iamgoofball** [comment]
the server and codebase are separate entities, but using the /tg/station servers as an example, they have an explicit rule against rolling a normal station job exclusively to greytide

**Comment by Helianthus00** [comment]
A config option would be nice but also I fully support this as-is.

**Comment by AtlasRules** [comment]
revolutionary idea: coders stop doing the admin's jobs and they do it themselves

wacky i know :trollface:

**Comment by AtlasRules** [comment]
> 
> 
> image
> extremely relevant meme

smh censoring the grey man, typical /tg/

**Comment by merrgear2** [comment]
Well atleas5 I don't play tg anymore to give a shit. But for people who wanna chill and just go do random shit this is fucked. People who constantly pick assistant are bored of the real jobs. Instead of making them play boring shit how about you make the jobs less boring? Plus assistants are flexible. If you ask them nicely to do a job for a few they will. Let people play whatever role they want in a 2d spessman game.

**Comment by boy2mantwicethefam** [comment]
I think I prefer "'vaccines cause autism' meme PR" era goofball than the current "I am triggered that one of the updates contained the number 1488 and make extremely questionable balance changes under an actively-hostile-to-the-codebase higher-up" era goofball
I hope that one day you will come to your senses, not because of this PR but because of the way you've been acting for the last few years. If your online friends do an instant 180 degree turn on you the moment you do something they don't like instead of acting like civilized human beings then you should reconsider your pals.

**Comment by FargothUr** [comment]
no this is fucking stupid, assistants should always exist as populace that gives security and medical purpose to exist, if you put them in actual departments, unless you're going to remove the job cap for every department then you will never have enough people for security and medical to have actual fun.

**Comment by SuperNovaa41** [comment]
> 
> 
> People really getting rilled up about a goof shit post.

if only it were a shitpost

**Comment by AtlasRules** [comment]
> 
> 
> > image
> > extremely relevant meme
> 
> smh censoring the grey man, typical /tg/
 "YOU WILL NOT POST GREY MAN MEME ON TERRIBLE GOOFBALL PR"
 https://tenor.com/view/1984-gif-19260546

**Comment by Twaticus** [comment]
_This comment was marked as spam._

**Comment by SamDDBS** [comment]
1984

**Comment by FargothUr** [comment]
Okay, since you actually hid my comment instead of actually addressing my criticisms in a mature manner, you clearly do not give a shit of what your players think, rather, you just think "my changes are just no matter what!" in that case, why do you even have a comment section? At least remove it so people can know how dishonest you are about handling any feedback that isn't sucking your toes

**Comment by FargothUr** [comment]
this is why a furry erp server is more populated than your shithole

**Comment by Iamgoofball** [comment]
Hi, I agree we need more job content. Feel free to PR it!

**Comment by tralezab** [comment]
walter white gif

**Comment by SamDDBS** [comment]
> 
> 
> Hi, I agree we need more job content. Feel free to PR it!

wait for someone to pr it before you nuke assistants

**Comment by FargothUr** [comment]
> 
> 
> Hi, I agree we need more job content. Feel free to PR it!

you seriously half assed this PR, you only removed the role yet not removed the job caps for departments

**Comment by FargothUr** [comment]
keep marking my actual points as spam, it wont make me wrong

**Comment by FargothUr** [comment]
jesus christ actually disable your comments if you disregard actual criticism you cannot debate, my god

**Comment by LemonInTheDark** [comment]
> What if all the job slots are taken?

Read the fucking pr title god damn it I don't care about this change but jesus christ

**Comment by FargothUr** [comment]
> 
> 
> Hi, I agree we need more job content. Feel free to PR it!

can you address my points instead of marking them as spam? thank you

**Comment by SuperNovaa41** [comment]
> 
> 
> jesus christ actually disable your comments if you disregard actual criticism you cannot debate, my god

please stop having a meltdown its not that serious

**Comment by bastardblasted** [comment]
> > jesus christ actually disable your comments if you disregard actual criticism you cannot debate, my god
> 
> please stop having a meltdown its not that serious

no keep having one its funny

**Comment by FargothUr** [comment]
> 
> 
> > What if all the job slots are taken?
> 
> Read the fucking pr title god damn it I don't care about this change but jesus christ

then this change really shouldn't exist, people are just going to be assistant anyways

**Comment by FargothUr** [comment]
das rite whitey, this change literally is bloat

**Comment by Bluedazzled** [comment]
after half the server on sybil greytided and a poor soldier lost to the permaban (stop jahir) the greytide has prevailed. hail the greytide

**Comment by tralezab** [comment]
image

**Comment by Knuff-les** [comment]
I love this change. beestation might start a couple more servers and NSV might be a daily thing if the surge of refugees kicks in right

**Comment by SuperNovaa41** [comment]
> 
> 
> I love this change. beestation might start a couple more servers and NSV might be a daily thing if the surge of refugees kicks in right

Closed

**Comment by Knuff-les** [comment]
> > I love this change. beestation might start a couple more servers and NSV might be a daily thing if the surge of refugees kicks in right
> 
> Closed

Sad, I just saw it

**Comment by optimumtact** [comment]
> I love this change. beestation might start a couple more servers and NSV might be a daily thing if the surge of refugees kicks in right

MRPstation

**Comment by actioninja** [comment]
Since Oranges is clearly not taking this seriously I'm taking over direction on this and allowing it to remain open.

**Comment by Burgermancoding** [comment]
People should be able to choose a job in which they don't have to put effort in and can dick around.

**Comment by tralezab** [comment]
maintainer battle incoming!

**Comment by ShibaInuLord** [comment]
oh god

**Comment by SplinterGP** [comment]
Please merge this PR, we shall end the greytide so colored tides in all the rainbow colors may flourish, such as the yellowtide(engineer) science tide(purple) cargotide(poop) medtide(white or blue)

**Comment by SuperNovaa41** [comment]
this is the end

**Comment by LemonInTheDark** [comment]
https://www.youtube.com/watch?v=5l4OEJ38aLY

**Comment by SilencedMP5A5** [comment]
literally crying rn

**Comment by ShibaInuLord** [comment]
instead of this why dont we add gimmick jobs like on goon

**Comment by SuperNovaa41** [comment]
> 
> 
> instead of this why dont we add gimmick jobs like on goon

Comment marked as spam.

**Comment by Psody-Mordheim** [comment]
based pr, delicious salt from the greytide

**Comment by InsaneRed** [comment]
GETOUTOFMYHEADGETOUTOFMYHEADGETOUTOFMYHEADGETOUTOFMYHEADGETOUTOFMYHEADGETOUTOFMYHEADGETOUTOFMYHEADGETOUTOFMYHEADGETOUTOFMYHEAD

**Comment by ShibaInuLord** [comment]
> 
> 
> > instead of this why dont we add gimmick jobs like on goon
> 
> Comment marked as spam.

i cant believe you would do this

**Comment by optimumtact** [comment]
>GETOUTOFMYHEADGETOUTOFMYHEADGETOUTOFMYHEADGETOUTOFMYHEADGETOUTOFMYHEADGETOUTOFMYHEADGETOUTOFMYHEADGETOUTOFMYHEADGETOUTOFMYHEAD

We have a code green

**Comment by LemonInTheDark** [comment]
> 
> 
> instead of this why dont we add gimmick jobs like on goon

Goon coders bully us enough as it is

**Comment by ShibaInuLord** [comment]
yeah you guys need to get good

**Comment by SamDDBS** [comment]
is that a s i m p i smell?

**Comment by maxymax13** [comment]
it's almost like they're not going to do a job anyways

**Comment by ShibaInuLord** [comment]
> 
> 
> is that a s i m p i smell?

no its goof

**Comment by GrayRachnid** [comment]
18 days late

**Comment by Faceslav** [comment]
are ya winning TG

**Comment by JoshAdamPowell** [comment]
I think this PR is probably good but maybe bad, can we test-merge it for a bit? Im mainly worried all the assistants will just be engies and assistants-but-with-insuls

**Comment by LemonInTheDark** [comment]
> 
> 
> I think this PR is probably good but maybe bad, can we test-merge it for a bit? Im mainly worried all the assistants will just be engies and assistants-but-with-insuls

This pr is both good and bad, it's balanced on a knife's edge, teetering over the abyss of decision

**Comment by HugoOdaX** [comment]
speedmerge plz

**Comment by ninjanomnom** [comment]
It's been two hours, I'll unlock this later when children go to bed

**Comment by LemonInTheDark** [comment]
image

**Comment by ninjanomnom** [comment]
You asked for this oranges, you don't get to kill what you brought into the world before it has even lived 24 hours. Reflect on your actions in this brief reprieve.

**Comment by optimumtact** [comment]
yes I can

**Comment by spookydonut** [comment]
h

**Comment by 81Denton** [comment]
On one hand, we shouldn't cater to burnt out admins that are too chicken shit to just ban the tiders and griefers they keep whining about
On the other hand, this PR infuriates the redditor

**Comment by FernandoJ8** [comment]
Optimum cease this. You're not playing fair.

**Comment by optimumtact** [comment]
I am above the law

**Comment by FernandoJ8** [comment]
You're not above, the law, you ARE the law, you fool!

**Comment by The-Sun-In-Splendour** [comment]
The tyranny of the fruits begin

**Comment by Kitsunemitsu** [comment]
🍿

**Comment by maxymax13** [comment]
image

**Comment by LemonInTheDark** [comment]
image

**Comment by BootlegBow** [comment]
i've never seen a pr closed and reopened so many times in 2 hours

**Comment by ShibaInuLord** [comment]
> 
> 
> image

stop using compact you nerd

**Comment by FernandoJ8** [comment]
This is what the internet was made for.
This is it.
This is what we were all destined to become.

**Comment by moocowswag** [comment]
Assistants bring a healthy amount of conflict to a round when dynamic fails to do so, and now their existence is bringing healthy conflict to github.

**Comment by LemonInTheDark** [comment]
> 
> 
> > image
> 
> stop using compact you nerd

no

**Comment by LemonInTheDark** [comment]
> 
> 
> Assistants bring a healthy amount of conflict to a round when dynamic fails to do so, and now their existence is bringing healthy conflict to github.

Meteors bring a healthy amount of conflict to a round when dynamic fails to do so, and now their existence is bringing healthy conflict to github.

**Comment by EOBGames** [comment]
I've seen more health in a palliative care ward than the discussion on this PR

**Comment by ninjanomnom** [comment]
Compact is the only tgstation officially endorsed discord mode and we will not provide technical support for those who use otherwise.

**Comment by maxymax13** [comment]
> 
> 
> > Assistants bring a healthy amount of conflict to a round when dynamic fails to do so, and now their existence is bringing healthy conflict to github.
> 
> Meteors bring a healthy amount of conflict to a round when dynamic fails to do so, and now their existence is bringing healthy conflict to github.

you can't baton a meteor

**Comment by interestingusernam3** [comment]
vore

**Comment by FernandoJ8** [comment]
> 
> 
> > > Assistants bring a healthy amount of conflict to a round when dynamic fails to do so, and now their existence is bringing healthy conflict to github.
> > 
> > 
> > Meteors bring a healthy amount of conflict to a round when dynamic fails to do so, and now their existence is bringing healthy conflict to github.
> 
> you can't baton a meteor

Try to stop me you filthy coward.

**Comment by ShibaInuLord** [comment]
> 
> 
> > > Assistants bring a healthy amount of conflict to a round when dynamic fails to do so, and now their existence is bringing healthy conflict to github.
> > 
> > 
> > Meteors bring a healthy amount of conflict to a round when dynamic fails to do so, and now their existence is bringing healthy conflict to github.
> 
> you can't baton a meteor

who told you that

**Comment by Crystaldude** [comment]
FINE ILL TAKE MY GREY TIDE SOME WHERE ELSE THEN

**Comment by InsaneRed** [comment]
https://www.youtube.com/watch?v=eBDi0iM2kcU

**Comment by Ziiro** [comment]
This is a horrible idea

Do it

**Comment by Ziiro** [comment]
Also from a code perspective you may want to make sure the "Be assistant if no other jobs available" option disappears once they're over the hours or something.

**Comment by Mothblocks** [comment]
i just woke up

**Comment by LemonInTheDark** [comment]
HERE WE GOOOOOOOOOOOOO

**Review by Kylerace — APPROVED** [review]

**Comment by theta006** [comment]
time for half the crew to be jobs they dont want to do

**Comment by FernandoJ8** [comment]
I KNEW WE COULD COUNT ON YOU.
THE HERO, THE UNDERDOG:
MOTHBLOCKS

**Comment by SamDDBS** [comment]
time for the jobs to be full of incompetent people who just wanted to have fun playing spessmans while not having any responsibilities

**Comment by Aether217** [comment]
> time for the jobs to be full of incompetent people who just wanted to have fun playing spessmans while not having any responsibilities

if only there were jobs like clown, mime, curator, chaplain, psychologist, etc... that can fuck around and do gimmicks....

**Comment by FernandoJ8** [comment]
> 
> 
> time for the jobs to be full of incompetent people who just wanted to have fun playing spessmans while not having any responsibilities

I shall cite words of a great man by the name of Ryder Perkins who, in his eternal wisdom, blessed us mortals with this blessed verse. It reads:
I sure do love job I sure do love this game, where the playstyle is to walk around saying nothing and just doing your job. Most fun thing ever. SS13, more like job simulator. I love job. Never do anything other than your job. Your job is your life. Doing anything other than your job is bad. Only job. Job good. No fun, only job.

Amen.

**Comment by optimumtact** [comment]
image

**Comment by Mothblocks** [comment]
> I sure do love job I sure do love this game, where the playstyle is to walk around saying nothing and just doing your job. Most fun thing ever. SS13, more like job simulator. I love job. Never do anything other than your job. Your job is your life. Doing anything other than your job is bad. Only job. Job good. No fun, only job.

i think theres more to this game but wahtever floats your boat man

**Comment by Learner10** [comment]
Why would you make a PR like this? Why do you hurt us like this? Anyhow, didn't Oranges make an intern PR for new players a year ago anyhow?

**Comment by mirrorcult** [comment]
seems like the underlying issue is that people would rather jack off and do nothing as a greyshirt than play job content

**Comment by Mothblocks** [comment]
We don't allow "jack off" on -TG- servers

**Comment by mirrorcult** [comment]
> 
> 
> We don't allow "jack off" on -TG- servers

image

**Comment by tralezab** [comment]
unironically buff sec to be better at catching tators since half of jacking off as assistant is just blasting traitors you find to dust, a lot of players would rather run grey than red for killing antags

**Comment by nightred** [comment]
This is a great plan, you never had todo your job, just do what ever the people that will keep the station running will do it regardless.

**Comment by Learner10** [comment]
> 
> 
> seems like the underlying issue is that people would rather jack off and do nothing as a greyshirt than play job content

I don't really see what's wrong with that. You'd be hard-pressed to win an argument that it's a bad thing to let a player choose a legally approved role that meshes with the attitude that they feel at the time and otherwise heightens their enjoyment of the game. People play games often to pass the time, and if people just happen to want to pass the time and sit along for the ride, I honestly don't see why it's a bad thing. Code and balance wise, the inclusion of assistants really change nothing, but their removal changes a lot. They're subject to the same problems other players are at risk of, but aren't going to get bitched at unlike a job role with a vital role.

If someone who knows engineering runs as an assistant, I mean sure, he 'could' help out in engineering if it goes bad because he's subject to the engine going down same as the rest of the folks, but that's no reason to make him do engineering, or deny a role FOR engineering to actually perform vital engineer duties just because he wanted to go build a project in space or maint. Forcing him to pick engineer or get in trouble for slacking off just spurs tyranical

All this does is

**Comment by carshalash** [comment]
People will start 'playing' bartender for low responsibilities if this goes through, that is my job and people will take muh rolls.

**Comment by ilikesaying** [comment]
>clowntide and mimetide start blocking the hallways and just making moving to different parts of the station more frustrating

**Comment by CRITAWAKETS** [comment]
> People will start 'playing' bartender for low responsibilities if this goes through, that is my job and people will take muh rolls.

nono, they'll play engineer and paramedic. Engineer especially has 0 responsabilities beyond existing and repairing things that you dont need to repair because da shuttl and they start with tools. Meanwhile paramedic has bootleg AA which is already enough to make it so that people have already switched from assistant to paramedic without this PR

**Comment by nianjiilical** [comment]
ill only accept this if clown becomes the new overflow role

in all seriousness i do worry about what effects not having a designated "just vibe and explore and chill" role will have but this is probably a meme pr anyway so who even knows

**Comment by ChungusGamer666** [comment]
> revolutionary idea: coders stop doing the admin's jobs and they do it themselves
> 
> wacky i know :trollface:

absolute skyrat opinion, admins are literally just a bandaid for bad code and game design

**Comment by necromanceranne** [comment]
> unironically buff sec to be better at catching tators since half of jacking off as assistant is just blasting traitors you find to dust, a lot of players would rather run grey than red for killing antags

They're strong enough. Like, REALLY strong. The issue is that they're red, not grey, so they have less of a chance to catch you by surprise. That's why people who want to fight play assistant, they can be wolves in sheeps clothing.

**Comment by Mothblocks** [comment]
We got Learner10 guys, that's what I was waiting for this whole time.

Thanks everyone who helped contribute to this, Learner10 looks so stupid right now lol

**Comment by ChungusGamer666** [comment]
>this pr was not merged
day ruined

**Comment by GrayRachnid** [comment]
>this pr
day ruined

**Comment by LetterN** [comment]
"hey let's remove assistants and force people to play other roles"

**Comment by Mothblocks** [comment]
okay

**Comment by LetterN** [comment]
that's it, i'm oranges now
image

**Comment by TheRealScarHomie** [comment]
guys just nerf everything an assistant does into the round like youve been doing instead???? I say next pr we give jumpsuits a slowdown because they are bulkier too, I think that would be a great TG pr, very skyrat 👍🏿

**Comment by guyman500** [comment]
"Viros out, Cant lock in assistant. Guess Ill log off"

**Comment by BowStache** [comment]
Jesus christ, this has to be a joke

**Pull request closed without merging**